src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json

```diff
@@ -1,16 +1,23 @@
 {
-    "version": "2.0",
-    "logging": {
-        "logLevel": {
-            "default": "Warning",
-            "Microsoft": "Warning",
-            "Function": "Information",
-            "GovUk.Education.ExploreEducationStatistics": "Information"
-        }
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
+      "samplingSettings": {
+        "isEnabled": true,
+        "excludedTypes": "Request"
```

Comment:
Yes this setting is based on this note in the docs. It's to prevent request logging from being sampled so all requests should be logged.

![image](https://github.com/dfe-analytical-services/explore-education-statistics/assets/4147126/b0bca874-f4aa-4e09-a347-7d429a766818)